hawkular-alerters/hawkular-elasticsearch-alerter/hawkular-elasticsearch-alerter-lib/src/main/java/org/hawkular/alerter/elasticsearch/ElasticsearchAlerter.java

```diff
@@ -69,7 +69,7 @@
  *   Documents fetched from Elasticsearch need a date field to indicate the timestamp.
  *   This timestamp will be used in the queries to fetch documents in interval basis.
  *
- *   If there is not defined a specific pattern under the trigger.context["timestamp.pattern"] it will follow the
+ *   If there is not defined a specific pattern under the trigger.context["timestamp_pattern"] it will follow the
```

Comment:
Because Elasticsearch 2.x version doesn't accept dot as part of field name.
And when storing a full alert, it might fail with context with dots.
It is fixed in upper versions but 2.x is the target used for OS deployments, so that's motivation.